cmd/vaultworker/res/configuration-docker.toml

```diff
@@ -24,6 +24,13 @@ scheme = "https"
 server = "edgex-vault"
 port = "8200"
 certpath = "v1/secret/edgex/pki/tls/edgex-kong"
+mongosecretpath = "v1/secret/edgex/mongodb"
```

Comment:
will address it in the future and being tracked in #75

Reply:
I'm worried about addressing this in the future as we have had issues where PR's have been merged where large issues like this are left as "future TODO's" then never gets done in time and so we are left with the original implementation which was incomplete and now needs to be supported. Since this is a new feature we  are adding into Fuji, we need to make sure this is implemented in a way that is maintainable and will be easy to not break backwards compatibility. 

To put it another way I can't approve this PR until issue #75 is resolved inside the PR or the code is refactored to not implement the keys at all in the configuration.toml specifically because of my concerns with merging half-implemented features

Reply:
ok I removed the current package that creates password dynamically and add log for future implementation.

Reply:
@tingyuz this still hasn't been resolved in this PR, your comment 
> ok I removed the current package that creates password dynamically and add log for future implementation.

 applies to #77, not #75